src/server/controller/plugin.ts

```diff
@@ -0,0 +1,56 @@
+import { PostgresGateway } from "../gateways/index";
+import { Trade } from "../types";
+// import memcached from 'memcached';
+
+export default class Controller {
+  db: PostgresGateway;
+
+  constructor() {
+    this.db = new PostgresGateway();
+  }
+
+  public async write(body: Trade): Promise<{}> {
```

Comment:
**Comment**: This should not be returning `Promise<{}>`.

Reply:
Updated types.